ansible-ipi-install/roles/installer/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
```

Comment:
shouldn't we use python3 ?

Reply:
Interesting. I didn't create this file. It was created for me when I initiated the creation of the ansible role using ansible-galaxy cmd.